templates/generic/rpi4_reference_vc4_dunfell.json

```diff
@@ -0,0 +1,225 @@
+                    "rw",
+                    "X-dobby.optional"
+                ]
+            }
+        ],
+        "envvar": [
+            "LD_PRELOAD=/usr/lib/libGLESv2.so.2:/usr/lib/libwayland-egl.so.1",
+            "XDG_RUNTIME_DIR=/tmp",
+            "RIALTO_SOCKET_PATH=/tmp/rialto-0"
+        ],
+        "devs": [
+            {
+                "type": "c",
+                "path": "/dev/vchiq",
+                "major": 243,
+                "minor": 0,
+                "access": "rw"
+            },
+            {
+                "type": "c",
+                "path": "/dev/snd/controlC0",
+                "major": 116,
+                "minor": 0,
+                "access": "rw"
+            },
+            {
+                "type": "c",
+                "path": "/dev/snd/seq",
+                "major": 116,
+                "minor": 1,
+                "access": "rw"
+            },
+            {
+                "type": "c",
+                "path": "/dev/snd/pcmC0D0p",
+                "major": 116,
+                "minor": 16,
+                "access": "rw"
+            },
+            {
+                "type": "c",
+                "path": "/dev/snd/pcmC0D1p",
+                "major": 116,
+                "minor": 17,
+                "access": "rw"
+            },
+            {
+                "type": "c",
+                "path": "/dev/snd/timer",
+                "major": 116,
+                "minor": 33,
+                "access": "rw"
+            },
+            {
+                "type": "c",
+                "path": "/dev/dri/card0",
+                "major": 226,
+                "minor": 0,
+                "access": "rw"
+            }
+        ],
+        "gfxLibs": [
+            {
+                "src": "/usr/lib/dri/vc4_dri.so",
+                "dst": "/usr/lib/dri/vc4_dri.so"
+            },
+            {
+                "src": "/usr/lib/libEGL.so.1",
+                "dst": "/usr/lib/libEGL.so"
+            },
+            {
+                "src": "/usr/lib/libEGL.so.1",
+                "dst": "/usr/lib/libEGL.so.1"
+            },
+            {
+                "src": "/usr/lib/libGLESv2.so.2",
+                "dst": "/usr/lib/libGLESv2.so"
+            },
+            {
+                "src": "/usr/lib/libGLESv2.so.2",
+                "dst": "/usr/lib/libGLESv2.so.2"
+            },
+            {
+                "src": "/usr/lib/libwayland-egl.so.1",
+                "dst": "/usr/lib/libwayland-egl.so"
+            },
+            {
+                "src": "/usr/lib/libwayland-egl.so.1",
+                "dst": "/usr/lib/libwayland-egl.so.0"
+            },
+            {
+                "src": "/usr/lib/libwayland-egl.so.1",
+                "dst": "/usr/lib/libwayland-egl.so.1"
+            },
+            {
+                "src": "/usr/lib/libessos.so.0",
+                "dst": "/usr/lib/libessos.so.0"
+            },
+            {
+                "src": "/usr/lib/dri",
```

Comment:
Do you really need to add the entire directory? We prefer specific libs in this section. Remark that one of the libs in there was already explicitly added: /usr/lib/dri/vc4_dri.so. If possible add specific dri libs here instead of the entire directory.

Reply:
I have added required libs for my application. Please review it.

Reply:
Changes fine after also renaming to rpi4_reference_dunfell (below). Thanks